packages/core/src/babel-plugin-adjust-imports.ts

```diff
@@ -64,6 +65,29 @@ type DefineExpressionPath = NodePath<CallExpression> & {
   };
 };
 
+export function isImportSyncExpression(path: NodePath<any>) {
```

Comment:
See `referencesImport` as used in https://github.com/embroider-build/embroider/blob/d5167811ef396d6cc0e0212f3f2edd44f9812cc3/packages/macros/src/babel/macros-babel-plugin.ts#L91 for a better way to implement `isImportSyncExpression`. Specifically, if somebody says 

```js
import { importSync as i } from "@embroider/macros"; 
i('foo')
```

we still want to detect that case.

Reply:
fixed